### ISSUE TITLE
About training the vgg-vd-16 model from scratch

### ISSUE THREAD

**Issue by qiulesun** [vedaldi/practical-cnn#26]
I want to train the vgg-vd-16 model from scratch, which parameter initialization method should I choose, gaussian or xavier or xavierimproved?

**Comment by qiulesun** [comment]
I use the batch-normalization,but the top1err decreased only from 0.99 to 0.91 after 5 epochs, and the convergence rate was slow.Is this normal?
